journal-of-investigative-dermatology.csl

```diff
@@ -1,227 +1,309 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-US">
-  <!-- This style was edited with the Visual CSL Editor (http://steveridout.com/csl/visualEditor/) -->
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Journal of Investigative Dermatology</title>
+    <title>Journal of investigative dermatology</title>
+    <title-short>J Invest Dermatol</title-short>
     <id>http://www.zotero.org/styles/journal-of-investigative-dermatology</id>
     <link href="http://www.zotero.org/styles/journal-of-investigative-dermatology" rel="self"/>
-    <link href="http://www.zotero.org/styles/ecology-letters" rel="template"/>
-    <link href="http://www.nature.com/jid/author_instructions.html" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://endnote.com/downloads/style/journal-investigative-dermatology" rel="documentation"/>
```

Comment:
please remove the link to the Ednote style -- both because it's not documentation and because their legal department is prone to making outrageous claims (i.e. I wouldn't put it past them to claim that using their style in any way violates their EULA)